src/shaders/_prelude_fog.fragment.glsl

```diff
@@ -22,25 +22,19 @@ float fog_opacity(vec3 position) {
     float start = u_fog_range.x;
     float end = u_fog_range.y;
 
-    // Apply a constant to push the function closer to 1.0 on the far end
-    // of the fog range, refer https://www.desmos.com/calculator/x5gopnb91a
-    const float exp_constant = 5.5;
-    float fog_falloff = exp(-exp_constant * (depth - start) / (end - start));
-
-    // Apply a power remove the C1 discontinuity at the near limit
-    const float fog_power = 2.0;
-    float opacity = pow(max((1.0 - fog_falloff) * u_fog_opacity, 0.0), fog_power);
-
-    // Clip to actually return 100% opacity at end
-    return min(1.0, opacity * 1.02);
+    // Fog falls off exponentially, but it also doesn't start at an arbitrary
+    // distance. So we opt for a sigmoid function that results in a very smooth
+    // onset. See: https://www.desmos.com/calculator/boumcg1dwo
+    // Fog power puts the fog at about 0.6% and 99.4% at near and far, respectively
+    // The multiplier puts the far limit at 1.0 since clipping depends on that
+    const float fog_pow = 10.0;
+    float fog_falloff = min(1.0, 1.00675 / (1.0 + exp(-fog_pow * ((depth - start) / (end - start) - 0.5))));
+
+    return fog_falloff * u_fog_opacity;
 }
 
 vec3 fog_apply(vec3 color, vec3 position) {
-    return gamma_mix(
```

Comment:
Before we go ahead with removing, I wanted to be sure that the screenshots are a fair comparison. If I were to singularize the bands and sample right at the limits, tops and bottoms don't match (in the whites).

Right tends to full white srgb(1,1,1) while left tends to srgb(0.8,0.8,0.8). If we were to use the same colors, would we still have that illusion of reducing the range?

<img width="161" alt="Screen Shot 2021-03-26 at 3 47 16 PM" src="https://user-images.githubusercontent.com/7061573/112700129-b3312a80-8e4a-11eb-80db-e9c5021b026b.png">